CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
+# 0.6.0
+
+* Enhancement - Renamed default version and branch to `mainline`.
+* Enhancement - Added support for [AWS SSO](https://aws.amazon.com/single-sign-on/) ([#229](https://github.com/awslabs/amazon-ecr-credential-helper/issues/229))
```

Comment:
I think this should be labelled as a feature

Reply:
Okay, I assume in that case would you agree the AWS SDK for Go revision would also be a feature. Once I address @kzys comment on what feature we enabled which I believe is the Apple M1 silicon.